### PR TITLE
DM-2111 - grafted changelog to y2024

### PR DESCRIPTION
**Pull request by jakubsobolewskisag**
lwm2m changelog

### CODE REVIEW

File: content/change-logs/device-management/lwm2m-agent-10-18-54-196-possible-to-disable-required-availability-mgmt.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    label: Improvement
+product_area: Device Management & Connectivity
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-ggH2M4hf3
+    label: lwm2m-agent
+ticket: DM-2111
+version: 10.18.54.196
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/lwm2m-agent-10-18-54-196-possible-to-disable-required-availability-mgmt.md:15]:
```suggestion
version: 10.18.540.195
```

File: content/change-logs/device-management/lwm2m-agent-10-18-54-196-possible-to-disable-required-availability-mgmt.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+change_type:
+  - value: change-2c7RdTdXo4
+    label: Improvement
+product_area: Device Management & Connectivity
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-ggH2M4hf3
+    label: lwm2m-agent
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/lwm2m-agent-10-18-54-196-possible-to-disable-required-availability-mgmt.md:13]:
```suggestion
  - value: tc-QHwMfWtBk7
    label: cumulocity
```

File: content/change-logs/device-management/lwm2m-agent-10-18-54-196-possible-to-disable-required-availability-mgmt.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+ticket: DM-2111
+version: 10.18.54.196
+---
+LWM2M agent manages required interval of LWM2M device automatically based on LWM2M registration lifetime. It's now possible to disable this behavior globally or for specific devices.
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/lwm2m-agent-10-18-54-196-possible-to-disable-required-availability-mgmt.md:17]: There is also user documentation part in the develop PR
https://github.com/SoftwareAG/c8y-docs/pull/2305/files

Comment (BeateRixen) [inline content/change-logs/device-management/lwm2m-agent-10-18-54-196-possible-to-disable-required-availability-mgmt.md:17]:
```suggestion
The LWM2M agent automatically manages the required interval of a LWM2M device based on the LWM2M registration lifetime. It is now possible to disable this behavior globally or for specific devices. 
```

Comment (BeateRixen) [reply content/change-logs/device-management/lwm2m-agent-10-18-54-196-possible-to-disable-required-availability-mgmt.md]: I assume that the user documentation changes need to be grafted as well. Will this happen here or in a separate PR?

Comment (jakubsobolewskisag) [reply content/change-logs/device-management/lwm2m-agent-10-18-54-196-possible-to-disable-required-availability-mgmt.md]: missed those - done!

**Review by Oezge-Akin — APPROVED** [review]

File: content/change-logs/device-management/lwm2m-agent-10-18-54-196-possible-to-disable-required-availability-mgmt.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-2111
+version: 10.18.540.195
+---
+The LWM2M agent automatically manages the required interval of a LWM2M device based on the LWM2M registration lifetime. It is now possible to disable this behavior globally or for specific devices which will result in a standard Cumulocity connection monitoring as described [in the documentation](https://cumulocity.com/docs/device-management-application/monitoring-and-controlling-devices/#to-monitor-the-connection-of-a-particular-device).
```
Comment (BeateRixen) [inline content/change-logs/device-management/lwm2m-agent-10-18-54-196-possible-to-disable-required-availability-mgmt.md:17]:
```suggestion
The LWM2M agent automatically manages the required interval of a LWM2M device based on the LWM2M registration lifetime. It is now possible to disable this behavior globally or for specific devices which results in a standard {{< product-c8y-iot >}} connection monitoring as described in [Monitoring and controlling devices](https://cumulocity.com/docs/device-management-application/monitoring-and-controlling-devices/#to-monitor-the-connection-of-a-particular-device). 
```

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/device-management/lwm2m-agent-10-18-54-196-possible-to-disable-required-availability-mgmt.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date:
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/lwm2m-agent-10-18-54-196-possible-to-disable-required-availability-mgmt.md:2]:
```suggestion
date: 2024-09-03
```

**Pull request merged**